src/test/resources/parameterwithannotations/input/org/testing/UnsolvedType.java

```diff
@@ -0,0 +1,3 @@
+package org.testing;
```

Comment:
Should this class have been added? The name suggests not.

(This is a tangent, but I've noticed that when Specimin crashes, there are often synthetic files left in the input folder. In a separate PR, it would be great if we could adjust Specimin so that it does all of its work in the output folder, instead, to avoid this problem. This isn't urgent, though, since it's only relevant when debugging a crash.)